### PR TITLE
Add thirdparty

### PR DESCRIPTION
**Pull request by asmaahassan90**

### CODE REVIEW

File: var/spack/repos/builtin/packages/openmpi/package.py [hunk 1]
```diff
@@ -196,15 +196,18 @@ class Openmpi(AutotoolsPackage):
     variant('thread_multiple', default=False,
             description='Enable MPI_THREAD_MULTIPLE support')
     variant('cuda', default=False, description='Enable CUDA support')
+    variant('hwloc', default='internal' , values=('internal', 'external'),
```
Comment (davydden) [inline var/spack/repos/builtin/packages/openmpi/package.py]: for consistency with other packages (see `hypre`), would you mind calling this `internal-hwloc` and make it boolean with default=False (as this is the previous behaviour)?

File: var/spack/repos/builtin/packages/py-thirdorder/package.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+# This is a template package file for Spack.  We've put "FIXME"
```
Comment (davydden) [inline var/spack/repos/builtin/packages/py-thirdorder/package.py]: remove that

File: var/spack/repos/builtin/packages/vasp/package.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+# This is a template package file for Spack.  We've put "FIXME"
```
Comment (davydden) [inline var/spack/repos/builtin/packages/vasp/package.py]: remove that.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,8 @@
 # <img src="https://cdn.rawgit.com/spack/spack/develop/share/spack/logo/spack-logo.svg" width="64" valign="middle" alt="Spack"/> Spack
 
-[![Build Status](https://travis-ci.org/spack/spack.svg?branch=develop)](https://travis-ci.org/spack/spack)
-[![codecov](https://codecov.io/gh/spack/spack/branch/develop/graph/badge.svg)](https://codecov.io/gh/spack/spack)
+[![Build Status](https://travis-ci.org/kaust-rc/spack.svg?branch=develop)](https://travis-ci.org/kaust-rc/spack)
```
Comment (davydden) [inline README.md]: those change are accidental, or?

**Comment by asmaahassan90** [comment]
I'm sorry this PR was intended to be to a fork, not the official repo. Thank you for your prompt response though.

**Comment by davydden** [comment]
@asmaahassan90 would you be willing to add some packages from this fork into Spack? Some users need at least VASP, other new packages won’t hurt either

**Pull request closed without merging**